modules/ROOT/pages/ext-analytics-elk.adoc

```diff
@@ -0,0 +1,174 @@
+= Analyze Business and API Data Using ELK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the Elastic Stack (ELK stack) to analyze the business data and API analytics generated by Anypoint Platform Private Cloud Edition (Anypoint Platform PCE). You can use Filebeat to process Anypoint Platform log files, insert them into an Elasticsearch database, and then analyze them with Kibana.
+
+The following example shows how API data appears in Kibana:
+
+image::kibana-example.png[]
```

Comment:
Is this image working? I can't see it in https://github.com/mulesoft/docs-private-cloud/blob/new-elk/modules/ROOT/pages/ext-analytics-elk.adoc

Reply:
@sebasampaoli do we need to update it?

Reply:
No need to update it, the image works well as an example.